src/Filesystem/F.php

```diff
@@ -598,15 +598,11 @@ public static function niceSize(
 	 */
 	public static function read(string $file): string|false
 	{
-		if (
-			is_readable($file) !== true &&
-			Str::startsWith($file, 'https://') !== true &&
-			Str::startsWith($file, 'http://') !== true
```

Comment:
`Issue`: We cannot remove these checks. Without them, it will be possible to do remote reads (which could be a vulnerability in internal networks). As Bruno had it in his original code example, these should be moved to the top of the method and return early.

Reply:
Your comment makes sense, but our old code does not really match that: 

```php
if (
	is_readable($file) !== true &&
	Str::startsWith($file, 'https://') !== true &&
	Str::startsWith($file, 'http://') !== true
) {
	return false;
}
```

Wouldn't it be correct to do this instead? 

```php
if (str_starts_with($file, 'https://') || str_starts_with($file, 'http://')) {
    return false;
}
```

Our old code does basically the opposite.

Reply:
I think it's totally worth to introduce this as a breaking change. I've rebased the PR to v5/develop. We already have the Remote class as a far better alternative to do remote requests and if you really want to keep it super basic, you can still use file_get_contents. I've added the check and also improved the tests.